word_game.rb

```diff
@@ -0,0 +1,160 @@
+require 'pry'
+require 'faker'
+
+class Game
+  def initialize
+    @word = GameData.new.easy_word.downcase
+    @user_choice = ''
+    @bad_array = []
+    @guess_string = ''
+    @change_split_word = []
+    @picture = GameData.new.picture
+  end
+  def level
+    print "would you like to play easy or hard?\n"
+    answer = gets.chomp.downcase
+    if answer == "easy"
+      print "I hope you know your Pokemon.\n"
+      @word = GameData.new.easy_word.downcase
+    else
+      print "There are no spaces in this Rick and Morty quote.\n"
+      @word = GameData.new.hard_word.downcase
+    end
+  end
+
+
+  def summary
+    puts "This is a summary the word is #{@word}"
+    puts "This is a check on #{@picture}"
+    puts "This is a check on #{@word}"
+  end
+
+  # Verbs
+  # input
+  def user_input
+    puts 'Choose a letter please'
+    @user_choice = gets.chomp.downcase
+
+
+    until @user_choice.length == 1
+      puts 'Hey! Pick a single letter.'
+      @user_choice = gets.chomp.downcase
+    end
+
+    while /\A\d+\z/.match(@user_choice)
+      puts 'Letters only please.'
+      @user_choice = gets.chomp.downcase
+    end
+    guess
+  end
+
+  # guess
+  def guess
+    if @word.include?(@user_choice)
+      print "Good guess that letter is in the word!\n"
```

Comment:
You've done a very good job of functional decomposition here - each function has one clear job, and they're neither too big nor too small.